components/tensorflow-notebook-image/releaser/components/workflows.libsonnet

```diff
@@ -0,0 +1,268 @@
+{
+  // TODO(https://github.com/ksonnet/ksonnet/issues/222): Taking namespace as an argument is a work around for the fact that ksonnet
+  // doesn't support automatically piping in the namespace from the environment to prototypes.
+
+  // convert a list of two items into a map representing an environment variable
+  listToMap:: function(v)
+    {
+      name: v[0],
+      value: v[1],
+    },
+
+  // Function to turn comma separated list of prow environment variables into a dictionary.
+  parseEnv:: function(v)
+    local pieces = std.split(v, ",");
+    if v != "" && std.length(pieces) > 0 then
+      std.map(
+        function(i) $.listToMap(std.split(i, "=")),
+        std.split(v, ",")
+      )
+    else [],
+
+
+  // Default parameters.
+  defaultParams:: {
+      bucket: "kubeflow-ci_temp",
+      commit: "master",
+      // Name of the secret containing GCP credentials.
+      gcpCredentialsSecretName: "kubeflow-testing-credentials",
+      name: "new9",
```

Comment:
The defaults are set suitable for building test images using our ci cluster. Our release cluster is more locked.

We want the defaults to be for the test cluster because we want the decision to use our release cluster to be an explicit action by a developer.